### DISCUSSION TITLE
Update Gravity Form use WP GraphQL Gravity Form functions

### DISCUSSION THREAD

**Discussion by reinis7** [WebDevStudios/nextjs-wordpress-starter#275] [Ideas] [answered]
WP GraphQL has extension to use Gravity Form as API that enables to use GraphQL functions in the Gravity Form integration with Frontend
https://www.wpgraphql.com/extenstion-plugins/wpgraphql-for-gravity-forms/

Would it be possible for us to integrate this in this platform to enhance WP GraphQL functions?
I have worked with your template for a month and I really like this and would like to contribute
Hope to discuss more in detail

Thanks

**Comment by gregrickaby** [comment, marked as answer]
@reinis7 Just a quick note that we are already using WP GraphQL Gravity Forms for the WordPress backend. See https://github.com/WebDevStudios/nextjs-starter-wordpress/blob/main/composer.json#L43
